### ISSUE TITLE
The console log shows the error.

### ISSUE THREAD

**Issue by CatzMR** [TavstalDev/TShop2#1]
I have installed TLibrary.dll normally and then it reported this error.
`ArgumentException: The specified path is not of a legal form (empty).
  at System.IO.Path.InsecureGetFullPath (System.String path) [0x00025] in <47fc8c70fa834cbf8141d7c1a7589125>:0
  at System.IO.Path.GetFullPath (System.String path) [0x00000] in <47fc8c70fa834cbf8141d7c1a7589125>:0
  at System.Diagnostics.FileVersionInfo.GetVersionInfo (System.String fileName) [0x00000] in <647615f3c1e6423a897a1a8f71c1fd3c>:0
  at Tavstal.TLibrary.Compatibility.PluginBase`1[PluginConfig]..cctor () [0x00057] in <026f8a66c2f44ad89e83fcbfdaa967e2>:0
Rethrow as TypeInitializationException: The type initializer for 'Tavstal.TLibrary.Compatibility.PluginBase`1' threw an exception.
  at Tavstal.TShop.TShop.Update () [0x00022] in <4d978f4fd8944fe885a774736ca73075>:0`

**Comment by CatzMR** [comment]
`I have installed TLibrary.dll normally and then it reported this error.ArgumentException: The specified path is not of a legal form (empty).
  at System.IO.Path.InsecureGetFullPath (System.String path) [0x00025] in <47fc8c70fa834cbf8141d7c1a7589125>:0
  at System.IO.Path.GetFullPath (System.String path) [0x00000] in <47fc8c70fa834cbf8141d7c1a7589125>:0
  at System.Diagnostics.FileVersionInfo.GetVersionInfo (System.String fileName) [0x00000] in <647615f3c1e6423a897a1a8f71c1fd3c>:0
  at Tavstal.TLibrary.Compatibility.PluginBase`1[PluginConfig]..cctor () [0x00057] in <026f8a66c2f44ad89e83fcbfdaa967e2>:0
Rethrow as TypeInitializationException: The type initializer for 'Tavstal.TLibrary.Compatibility.PluginBase`1' threw an exception.
  at Tavstal.TShop.TShop..ctor () [0x00000] in <4d978f4fd8944fe885a774736ca73075>:0
UnityEngine.GameObject:Internal_AddComponentWithType(GameObject, Type)
UnityEngine.GameObject:AddComponent(Type)
UnityEngine.GameObject:.ctor(String, Type[])
Rocket.Core.Plugins.RocketPluginManager:Rocket.Core.Plugins.RocketPluginManager.loadPlugins_Patch1(RocketPluginManager)
Rocket.Core.Plugins.RocketPluginManager:Start()

NullReferenceException: Object reference not set to an instance of an object
  at Rocket.Core.Plugins.RocketPlugin.LoadPlugin () [0x00017] in <2b8ee4b602174810b7294320f1aecbc4>:0
  at Rocket.Core.Plugins.RocketPlugin.OnEnable () [0x00000] in <2b8ee4b602174810b7294320f1aecbc4>:0
UnityEngine.GameObject:Internal_AddComponentWithType(GameObject, Type)
UnityEngine.GameObject:AddComponent(Type)
UnityEngine.GameObject:.ctor(String, Type[])
Rocket.Core.Plugins.RocketPluginManager:Rocket.Core.Plugins.RocketPluginManager.loadPlugins_Patch1(RocketPluginManager)
Rocket.Core.Plugins.RocketPluginManager:Start()`

**Comment by TavstalDev** [comment]
Fixed by the [4.0.8786.25897](https://github.com/TavstalDev/TShop2/releases/tag/4.0.8786.25897) release.

Reason of the error:
- TLibrary's FileVersionInfo.
- Missing libraries

**Comment by CatzMR** [comment]
Purchases were not possible on the UI and a new error popped up on the console.
```
[TShop] [EXCEPTION] > Error in UEventHandler -> OnButtonClick(bt_tshop_basket#buy):
[TShop] [ERROR] > System.NullReferenceException: Object reference not set to an instance of an object
  at Tavstal.TShop.Compability.Hooks.UconomyHook.GetBalanceAsync (Steamworks.CSteamID player, Tavstal.TLibrary.Compatibility.Economy.EPaymentMethod method) [0x0003f] in <116d602619744bf3b363c48ef3e0d31b>:0
  at Tavstal.TShop.Handlers.UnturnedEventHandler.Event_OnButtonClick (SDG.Unturned.Player player, System.String button) [0x00e5e] in <116d602619744bf3b363c48ef3e0d31b>:0
```
